Friendly-plans/app/src/androidTest/java/pg/autyzm/friendly_plans/resource/ChildViewClicker.java

```diff
@@ -0,0 +1,31 @@
+package pg.autyzm.friendly_plans.resource;
+
```

Comment:
The resource concept is provided by JUnit to declare actions to be done before and after the tests that are using this rule. (some info https://carlosbecker.com/posts/junit-rules/) so the package resource is prepared to contains all the rules/externalResources for junit. I think this class would match better in its own folder, prepared for classes like that, eg. view_operations or view_actions. I would also recommend to change name of that class, instead of ChildViewClicker it could be ViewClicker, because this class is used for both children and plans list. Could you please change it?